sal_interfaces/Dome/Dome_Telemetry.xml

```diff
@@ -1,397 +1,389 @@
-        <Units>A</Units>
-        <Count>2</Count>
-    </item>
-    <item>
-        <EFDB_Name>driveTempActual</EFDB_Name>
-        <Description>Temperature of light/wind screen drives/motors</Description>
-        <IDL_Type>float</IDL_Type>
-        <Units>deg_C</Units>
-        <Count>2</Count>
-    </item>
-    <item>
-        <EFDB_Name>encoderHeadRaw</EFDB_Name>
-        <Description>Raw value of light/wind screen encoders</Description>
-        <IDL_Type>float</IDL_Type>
-        <Units>unitless</Units>
-        <Count>2</Count>
-    </item>
-    <item>
-        <EFDB_Name>encoderHeadCalibrated</EFDB_Name>
-        <Description>Calibrated value of light/wind screen encoders</Description>
-        <IDL_Type>float</IDL_Type>
-        <Units>deg</Units>
-        <Count>2</Count>
-    </item>
-    <item>
-        <EFDB_Name>resolverRaw</EFDB_Name>
-        <Description>Raw value of light/wind screen resolvers</Description>
-        <IDL_Type>float</IDL_Type>
-        <Units>unitless</Units>
-        <Count>2</Count>
-    </item>
-    <item>
-        <EFDB_Name>resolverCalibrated</EFDB_Name>
-        <Description>Calibrated value of light/wind screen resolvers</Description>
-        <IDL_Type>float</IDL_Type>
-        <Units>deg</Units>
-        <Count>2</Count>
-    </item>
-    <item>
-        <EFDB_Name>powerAbsortion</EFDB_Name>
-        <Description>Current total power absorption</Description>
-        <IDL_Type>float</IDL_Type>
-        <Units>W</Units>
-        <Count>1</Count>
-    </item>
-</SALTelemetry>
-<SALTelemetry>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.7.1</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_domeMONCS_status</EFDB_Topic>
-    <item>
-        <EFDB_Name>data</EFDB_Name>
-        <Description>Safety monitoring</Description>
-        <IDL_Type>float</IDL_Type>
-        <Units>unitless</Units>
-        <Count>16</Count>
-    </item>
-</SALTelemetry>
-<SALTelemetry>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.7.1</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_domeTHCS_status</EFDB_Topic>
-    <item>
-        <EFDB_Name>data</EFDB_Name>
-        <Description>Temperature of heat generating units</Description>
-        <IDL_Type>float</IDL_Type>
-        <Units>deg_C</Units>
-        <Count>16</Count>
-    </item>
-</SALTelemetry>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+    <SALTelemetry>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_azimuth</EFDB_Topic>
+        <Description>Status of the azimuth axis and drives.</Description>
+        <item>
+            <EFDB_Name>positionActual</EFDB_Name>
+            <Description>Measured azimuth axis position.</Description>
+            <IDL_Type>float</IDL_Type>
+            <Units>deg</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>positionCommand</EFDB_Name>

```

Comment:
Should this be positionCommanded? Same for all other occurrences of Command in the rest of this file.

Reply:
OK. I'll make that change.